tools/Google.Cloud.Tools.ReleaseManager.IntegrationTests/ContainerCommands/DockerCommandTest.cs

```diff
@@ -62,6 +61,17 @@ public void DockerTest(string directory)
             var path = Path.Combine(outputDirectory, expectedAbsent);
             Assert.False(Path.Exists(path), $"Path {path} should not exist");
         }
+        var expectedDirectory = Path.Combine(outputDirectory, "expected");
+        if (Directory.Exists(expectedDirectory))
+        {
+            foreach (var expectedFilePath in Directory.GetFiles(expectedDirectory, "*.*", SearchOption.AllDirectories))
+            {
+                var actualFilePath = Path.Combine(outputDirectory, Path.GetRelativePath(expectedDirectory, expectedFilePath));
+                var expectedContent = File.ReadAllText(expectedFilePath);
+                var actualContent = File.ReadAllText(actualFilePath);
+                Assert.Equal(expectedContent, actualContent);
```

Comment:
In a follow-up, I suspect we'll want to make the failure show what files were being compared - but that can definitely come later.